src/PrestaShopBundle/Resources/config/services/core/form/choice_provider.yml

```diff
@@ -171,3 +176,13 @@ services:
     arguments:
       - '@request_stack'
       - '@prestashop.adapter.meta.data_provider'
+
+  prestashop.core.form.choice_provider.accessible_tab:
+    class: 'PrestaShop\PrestaShop\Core\Form\ChoiceProvider\TabChoiceProvider'
+    arguments:
+      - '@=service("prestashop.adapter.data_provider.tab").getViewableTabsForContextEmployee(service("prestashop.adapter.legacy.context").getLanguage().id)'
+
+  prestashop.core.form.choice_provider.profile:
+    class: 'PrestaShop\PrestaShop\Core\Form\ChoiceProvider\ProfileChoiceProvider'
+    arguments:
+      - '@=service("prestashop.adapter.data_provider.profile").getProfiles(service("prestashop.adapter.legacy.context").getLanguage().id)'
```

Comment:
Blank end line too

Reply:
there is a blank line, same comment as above.